Enhancing cost management setup language (#7517)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/cost-management/cost-management.md

Configuring the cost management features requires both a connection and a user component:

> - **[Connection setup](#connection-setup):** Set up the credentials used to access the data warehouse information. You can use the connection that has your main account identifier (name or id). Only one unique [connection](/docs/cloud/connect-data-platform/about-connections#connection-management) per warehouse needs to have the credentials configured. If you have two or more connections that reference the same account identifier, you only need to add platform metadata credentials to one of them.

Author: ```suggestion
- **[Connection setup](#connection-setup):** Set up the credentials used to access the data warehouse information. You can use the connection that has your main account identifier (name or ID). Only one unique [connection](/docs/cloud/connect-data-platform/about-connections#connection-management) per warehouse needs to have the credentials configured. If you have two or more connections that reference the same account identifier, you only need to add platform metadata credentials to one of them.
```

> - **[Connection setup](#connection-setup):** Set up the credentials used to access the data warehouse information. You can use the connection that has your main account identifier (name or ID). Only one unique [connection](/docs/cloud/connect-data-platform/about-connections#connection-management) per warehouse needs to have the credentials configured. If you have two or more connections that reference the same account identifier, you only need to add platform metadata credentials to one of them.

Reviewer: ```suggestion
- **[Connection setup](#connection-setup):** Set up the credentials that allow dbt to access your data warehouse metadata. Use the connection associated with your main account identifier (name or ID) to enable features like cost monitoring and usage insights. Only one unique [connection](/docs/cloud/connect-data-platform/about-connections#connection-management) per warehouse needs these credentials configured. If you have multiple connections reference the same account identifier, you only need to add platform metadata credentials to one of them.
```

Author: I want to leave that first sentence as-is to be super clear that these credentials are being used by our platform to get that info (rather than "allowing" something) I think that's an important distinction for security folks looking in. Same for the third line. 

```suggestion
- **[Connection setup](#connection-setup):** Set up the credentials used to access the data warehouse information. Use the connection associated with your main account identifier (name or ID). Only one unique [connection](/docs/cloud/connect-data-platform/about-connections#connection-management) per warehouse needs to have the credentials configured. If you have multiple connections that reference the same account identifier, you only need to add platform metadata credentials to one of them.
```

> - **[Provision user access](#provision-user-access):** Add new permissions to users and/or groups to regulate access to the dashboard

Reviewer: ```suggestion
- **[Provision user access](#provision-user-access):** Add new permissions to users and/or groups to regulate access to the dashboard.
```
